### PR TITLE
Update changelog.md to mention new assert with message feature

### PR DESCRIPTION
**Pull request by mit-mit**
Fixes https://github.com/dart-lang/sdk/issues/28134

### CODE REVIEW

**Comment by mit-mit** [comment]
@lrhn @floitschG can you review, please?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,16 +1,35 @@
 ## 1.22.0
 
+### Language
+
+  * The `assert()` statement has been expanded to support an optional second
+    `message` argument (SDK issue [27342](https://github.com/dart-lang/sdk/issues/27342)).
+
+    The message is display if the assert fails. It can be any object, and it
```
Comment (floitschG) [inline CHANGELOG.md]: displayed.

**Comment by floitschG** [comment]
LGTM (with one typo).

**Comment by mit-mit** [comment]
Thanks, typo fixed

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,16 +1,35 @@
 ## 1.22.0
 
+### Language
+
+  * The `assert()` statement has been expanded to support an optional second
+    `message` argument (SDK issue [27342](https://github.com/dart-lang/sdk/issues/27342)).
+
```
Comment (lrhn) [inline CHANGELOG.md]: Maybe use https://github.com/dart-lang/sdk/issues/24213 as the issue number.

**Pull request merged**